pkgs/os-specific/linux/bpftrace/default.nix

```diff
@@ -4,12 +4,20 @@
 , libelf, libbfd, libbpf, libopcodes, bcc
 , cereal, asciidoctor
 , nixosTests
+, util-linux
 }:
 
 stdenv.mkDerivation rec {
   pname = "bpftrace";
   version = "0.14.1";
 
+  # Rename *.8 to *.bt.8 to avoid conflicts with original bcc programs
+  # Cherry-picked from merged PR, remove this hook on next update
+  # https://github.com/iovisor/bpftrace/pull/2242
+  postUnpack = ''
+    ${util-linux}/bin/rename .8 .bt.8 "$sourceRoot"/man/man8/*.8
```

Comment:
```suggestion
    rename .8 .bt.8 "$sourceRoot"/man/man8/*.8
```

Reply:
Why?

This makes clear why the `util-linux` dependency was introduced.

This line will be removed with the next update, anyway.

There are multiple `rename` commands and omitting the package leaves readers in the dark as to which one is required:
```
nix-shell --pure -p hello --command rename
The program 'rename' is not in your PATH. It is provided by several packages.
You can make it available in an ephemeral shell by typing one of the following:
  nix-shell -p file-rename
  nix-shell -p libuuid
  nix-shell -p rename
  nix-shell -p util-linux
```

Reply:
It is already in nativeBuildInputs. We don't need both.

Reply:
The nativeBuildInputs variant also works when you cross-compile this packages, which is not the case when you prefixing it force-fully.

Reply:
Oh I see, thank make sense, thanks.